mesonbuild/interpreter.py

```diff
@@ -2448,7 +2449,7 @@ def validate_arguments(self, args, argcount, arg_types):
                 if not isinstance(actual, wanted):
                     raise InvalidArguments('Incorrect argument type.')
 
-    @FeatureNewKwargs('run_command', '0.50.0', ['env'])
+    @FeatureNewKwargs('run_command', '0.50.0', ['env', 'cwd'])
```

Comment:
The feature is new to 0.54.0, not 0.50.0, so add a new decorator rather than appending to an old one.